integration_tests/integration/createLicence.cy.ts

```diff
@@ -51,7 +51,7 @@ context('Create a licence', () => {
       const additionalConditionsInputPage = additionalConditionsPage
         .selectCondition('5db26ab3-9b6f-4bee-b2aa-53aa3f3be7dd')
         .selectCondition('fce34fb2-02f4-4eb0-9b8d-d091e11451fa')
-        .selectCondition('89e656ec-77e8-4832-acc4-6ec05d3e9a98')
+        .selectCondition('df3f08a8-4ae0-41fe-b3bc-d0be1fd2d8aa')
```

Comment:
*Comment*: 89e656ec has been removed (split into two) and df3f08a8 is one of the replacement conditions